content/applications/inventory_and_mrp/inventory/management/warehouses/use_locations.rst

```diff
@@ -0,0 +1,58 @@
+=================================
+Use locations to manage inventory
+=================================
+
+A *location* is a specific space within a warehouse. This can be a shelf, room, aisle, etc. There
+are three types of locations in Odoo:
+
+- *Physical locations* are spaces within a warehouse owned by the user's company. These can be a
+  an area where items are stored like an aisle or shelf, or an area where operations take place,
+  like loading and unloading bays.
+- *Partner locations* are the same as physical locations except that they exist within the warehouse
+  of a customer or vendor.
+- *Virtual locations* are locations that do not exist physically, but where items that are not in
+  inventory can be placed. These can be items that have not yet entered inventory, like products
+  that are on the way to a warehouse, or items that are no longer in inventory due to loss or other
+  factors.
+
+.. important::
+   In order to use locations, the :guilabel:`Storage Locations` setting must be enabled from the
+   :menuselection:`Inventory` module by selecting :menuselection:`Configuration --> Settings` and
+   clicking the checkbox under the :guilabel:`Warehouse` heading.
+
+Create a new location inside a warehouse
+========================================
+
+Starting from the :menuselection:`Inventory` module, select :menuselection:`Configuration -->
+Locations --> Create`. The new location form can then be configured as follows:
+
+- :guilabel:`Location Name`: the name that will be used to reference the location
+- :guilabel:`Parent Location`: the location or warehouse that the new location exists within
+- :guilabel:`Location Type`: choose the category that the location belongs to
+- :guilabel:`Company`: the company that owns the warehouse that the location is inside of
+- :guilabel:`Is a Scrap Location?`: check this box to allow for scrapped/damaged goods to be stored
+  in this location
+- :guilabel:`Is a Return Location?`: check this box to allow products to be returned to this
+  location
+- :guilabel:`Barcode`: the barcode number assigned to the location
+- :guilabel:`Removal Strategy`: the strategy for how items should be taken from inventory
```

Comment:
It might be good to link to the Remoal Strategy doc here:
```suggestion
- :guilabel:`Removal Strategy`: the :doc:`strategy
  </applications/inventory_and_mrp/inventory/routes/strategies/removal>` for how items should be
  taken from inventory
```